[bug][allure-mocha] runtime part isn't publiched to npm

**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior:
1. Install latest allure-mocha:2.0.0-beta.13
Use code
```
const runtime = require('allure-mocha/runtime');

describe('foo', () => {
  it('bar', () => {
    runtime.allure.epic("zopa")
  });
});
```

**Expected behavior**
Report files generated

**Actual behavior**
```
Error: Cannot find module 'allure-mocha/runtime'
Require stack:
- /home/bo/temp/123/test/index.spec.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
```

**Reason**

runtime.js isn't published to npm due changes https://github.com/allure-framework/allure-js/commit/db59be8ce144ce0d645a79165f1f25aa6c959c83#diff-f11d188b6edc479557088100cb599d1bbdea69d768f9a03361e48b80b34c3211R75

--- a/packages/allure-mocha/package.json
+++ b/packages/allure-mocha/package.json
@@ -73,6 +73,8 @@
     "temp-dir": "./out/.nyc"
   },
   "files": [
-    "dist"
+    "dist",
+    "runtime.js",
+    "runtime.d.ts"
   ]
 }

